### ISSUE TITLE
Add support for IntelliJ IDEA 2024.2

### ISSUE THREAD

**Issue by andygoossens** [wicketforge/wicketforge#200]
IntelliJ IDEA 2024.2 has been released. Please make the WicketForge plugin compatible.

Thanks.

**Comment by truelars** [comment]
Would be nice if @bamarc could accept the PR, as it is a trivial change.

In the meantime, you can always patch the plugin yourself. Just download [release 6.0.10](https://plugins.jetbrains.com/plugin/download?rel=true&updateId=522387) to a temporary folder and in that folder do the following:

```
# unzip plug-in
unzip WicketForge-6.0.10.zip

# unzip jar
cd WicketForge/lib
unzip instrumented-WicketForge-6.0.10.jar META-INF/plugin.xml

# replace version restriction
sed -i -e "s/241\./242\./g" META-INF/plugin.xml

# update jar
zip -u instrumented-WicketForge-6.0.10.jar META-INF/plugin.xml

# update plug-in
cd ../..
zip -u WicketForge-6.0.10.zip WicketForge/lib/instrumented-WicketForge-6.0.10.jar
```

Now you can use Settings -> Plugins -> Install From Disk with the updated zip file.

**Comment by bamarc** [comment]
Sorry for the wait, i've been super busy lately and am not using the plugin myself anymore. 

The fix to just up the version numbers will not fix the problem as Jetbrains changed the way the config has to be written in the newer versions of the intellij-platform plugin. I've re-writen most of the config to fit the new style and will publish a new version later today.

Also i'll up the supported version to infinity for now so the plugin can be used till it breaks completely or someone fixes it.

**Comment by bamarc** [comment]
I've published a new version 6.0.11 which includes a new maximum version of 283 which should provide support till something major breaks. 

Sorry again for the delay.
